ansible/roles/jenkinsslave/tasks/main.yml

```diff
@@ -34,6 +34,7 @@
     owner={{ jenkinsuser }}
     group={{ jenkinsuser }}
     mode=0700
+    remote_src=True
```

Comment:
I think the original intention (@aleksandra-tarkowska) was that keys would be copied from a _local_ directory. Copying from a `remote_src` would only work if the key has already been copied in some other way, which in general won't be the case.